src/PrestaShopBundle/Form/Admin/Sell/Customer/CustomerType.php

```diff
@@ -123,6 +133,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         $maxLength = $this->configuration->get(PasswordPolicyConfiguration::CONFIGURATION_MAXIMUM_LENGTH);
         $minLength = $this->configuration->get(PasswordPolicyConfiguration::CONFIGURATION_MINIMUM_LENGTH);
 
+        if ($options['show_guest_field'] === true) {
+            $builder
+                ->add('is_guest', SwitchType::class, [
+                    'label' => $this->trans('Register as guest', 'Admin.Global'),
+                    'help' => $this->trans(
+                        'Adds a one-time guest customer with no password.',
```

Comment:
```suggestion
                        'The user won't have a password to sign in to your store unless they convert their guest account into a regular one.',
```

Reply:
wdyt @Hlavtox ? I edited the wording for clarity.

Reply:
What about this, it's shorter.

`Guests don't have a password and can't log in to your store, unless they are converted to a regular customer.`

Reply:
I think we should keep the wording I suggested, a double negation isn't great in a wording. 
Besides, I believe using "guests (...) are converted" is vague. (You don't "convert" people? 🧐). This help text needs to be as clear as possible. I will edit my suggestion, let me know what you think.

Reply:
@l-delin I don't like it, because:
- We don't call customers users, users are if anywhere, in the backoffice.
- `Won't need a password to sign in` - nonsense, they **don't sign in** at all, they `don't have` the password.
- I checked it and we should use "transform" wording - that's used in both FO and BO.

What about this:
`Guests don't have a password and can't log into your store, unless their account is transformed to a customer account.`

Reply:
@l-delin I like this version, but I would keep the educational part where we explain what it means to be a Guest.

Reply:
`Visitors can order as guests without creating an account. At any time, you can turn your guests into customers.` ?

Reply:
LGTM :) wdyt @Hlavtox ?

Reply:
@l-delin now that I think more about it, there might be confusion because this information is visible in the back office, and there's already a group called "Visitors", etc. I think it's good to inform merchants about the actual consequences of adding a guest account, and the truth is that they won't have the password set, and they cannot log in as a regular customer does.

Reply:
@l-delin Sorry Lea but the wording make no sense. I am using Prestashop for 8 years now, fixed the guest ordering, guest transformation, guest groups and now the guest creation and the wording just doesnt say what it should.

Why the hell should I see `Visitors can order as guests without creating an account. At any time, you can turn your guests into customers.` when I create a guest in a backoffice? Guests aren't visitors, they cannot order.

The wording should say that the Guest doesnt have a password, that it's a one time customer for single use and it can be later transformed.

`Guests don't have a password and can't log into your store, unless their account is transformed to a customer account.` says it all and it's short.

Reply:
UPDATE: We have come with a good wording with @kpodemski. Our proposal, it's longer, but so far the best:

`Guests are quickly made customers with no password, who don't have access to the privileges of the registered ones. You can create as many guests as needed using the same email. This feature could be helpful if you take phone call orders and insert them into the system.`

Reply:
Alright. Then, let's go with the shorter version. If there needs to be any precision, it'll be added to the doc when the feature is released. The shorter version is better as this is a help text and we don't want it to be too long.

So:
`A quick customer with no password, who doesn't have access to the privileges of the registered one. You can create as many guests as needed using the same email. It could be helpful if you take phone call orders.`